components/webui/imports/ui/IngestView/panels/IngestionJobs/IngestionJobRow.jsx

```diff
@@ -75,13 +91,19 @@ const IngestionJobRow = ({job}) => {
                 {job._id}
             </td>
             <td className={"text-end"}>
-                {speedText}
+                <PlaceholderText
+                    isAlwaysVisible={job.status !== COMPRESSION_JOB_STATUS.FAILED}
```

Comment:
Maybe we could deduplicate `job.status !== COMPRESSION_JOB_STATUS.FAILED` into a variable?